src/main/kotlin/ch/bfh/habits/services/HabitService.kt

```diff
@@ -51,9 +51,6 @@ class HabitService(private val habitDAO: HabitDAO) {
 
     @Transactional
     fun updateHabitById(id: Long, habitDTO: HabitDTO) {
-        if (habitDTO.id != id) {
```

Comment:
We do not need this?

Reply:
updating a habit is not working at the moment.
The UI only sends the id in the path and not in the body of the request.
So we can discuss whether to send the ID inside the body or remove it in the backend.